Example-iOS/Podfile

```diff
@@ -0,0 +1,9 @@
+source 'https://github.com/CocoaPods/Specs.git'
+
+target 'VOKBenkode', :exclusive => true do
```

Comment:
Yeah, it should be.  Like I said, I just went with what `pod lib create` set up for me.  Want me to try changing it?